packages/shared/src/components/cards/common/PostCardFooter.tsx

```diff
@@ -12,29 +12,35 @@ interface PostCardFooterProps extends CommonCardCoverProps {
   openNewTab: boolean;
   post: Post;
   className: PostCardFooterClassName;
+  eagerLoadImage?: boolean;
 }
 
 export const PostCardFooter = ({
-  post,
   className,
+  eagerLoadImage = false,
   onShare,
+  post,
 }: PostCardFooterProps): ReactElement => {
   const isVideoType = isVideoPost(post);
+  const imageLoadingProps: ComponentProps<'img'> = {
+    loading: eagerLoadImage ? null : 'lazy',
+    fetchPriority: eagerLoadImage ? 'high' : 'auto',
+  };
```

Comment:
Does it make sense to move this as a common func that gets eagerLoadImage and returns the values, so it can be used in all cards?

Reply:
If we want to use a general logic we should add it to <Image /> component, but I don't know how widely we should apply this changes. Wdyt?

Reply:
Well it's already being used in 7 files already that's why I thought maybe would be good

Reply:
@nensidosari updated! ✅